lambda_functions/execute_ssh_command_js/handler.js

```diff
@@ -61,18 +61,26 @@ exports.execute_ssh_command = (event, context, callback) => {
 
             let command = create_execution_string(event);
 
-	        ssh.exec(command, {
+	        ssh
+	           .exec(command, {
                      exit: (code, stdout, stderr) => {
-                        if (stderr) {
-                                       console.log(`STDERR: ${stderr}`);
-                                       //  Return error with error information back to the caller
-                                       callback(`Failed to execute SSH command, ${stderr}`);
+                        if (code == 0) {
+                                         console.log(`Executing: ${command}`);
+                                         console.log(`STDOUT: ${stdout}`);
+                                         console.log(`SSH returncode: ${code}`);
+                                         const response = { statusCode: 0, body: 'SSH command executed.' };
+                                         // Return success with information back to the caller
+                                         callback(null, response);
+                        } else if (code == 1) {
```

Comment:
I don't see any reason to check for `code == 1`. If we just keep it simple, `0` means it completed successfully, anything else means there was some sort of error.

Reply:
Thanks Damien, for the comments. 
I will separate out deaenv_* raijin scripts by appending testing in front of the directory name. Is that ok?

I thought code == 1 would help us fix any raijin script errors or invalid command line options or malformed configuration. I will fix this one in the handler script to include only zero or not zero.

Reply:
Sounds good.

I think there's so many possibly error codes that so long as we're logging code/stderr/stdout we'll be able to work out what's going on.